docs/tutorial/multi_car_paint.ipynb

```diff
@@ -0,0 +1,541 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "plt.plot(cost_history)\n",
+    "plt.xlabel(\"Iteration\")\n",
+    "plt.ylabel(\"Cost\")\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we have obtained the QAOA parameters. Let's use them for sampling"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {
+    "id": "QjdU1QW_6X8N"
+   },
+   "outputs": [],
+   "source": [
+    "# Run Optimized QAOA circuit\n",
+    "sampler = qk_pr.StatevectorSampler()\n",
+    "qk_circuit.measure_all()\n",
+    "job = sampler.run([(qk_circuit, result.x)], shots=1000)\n",
+    "job_result = job.result()[0]\n",
+    "qaoa_counts = job_result.data[\"meas\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Evaluating the Results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "One can determine the optimal solution for the painting order."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "_ddpxjij6dv0",
+    "outputId": "aff5f8a9-537b-4b29-b02c-427eb254253a"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The order of car intake:  [1, 1, 0, 2, 3, 0, 3, 2]\n",
```

Comment:
This way to show the result is hard to understand the meaning, so please make more pretty plot. such as using emoji like 🚗 somothing